sdk-extension/opentelemetry-sdk-extension-aws/src/opentelemetry/sdk/extension/aws/resource/__init__.py

```diff
@@ -0,0 +1,31 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from opentelemetry.sdk.extension.aws.resource._lambda import (
```

Comment:
why does lambda module name starts with "_" and other don't?

Reply:
I was pretty shocked by that too 😝 It turns out because `lambda` is reserved keyword in python, it can't be used in imports either! So it has to have something different and I thought this was the least intrusive naming (I've seen it used in other Python programs that need to use that word).